app/views/companies/index.html.slim

```diff
@@ -0,0 +1,11 @@
+.container
+  .section
+    - @companies.group_by(&:supporter?).reverse_each do |supporter, companies|
```

Comment:
Yeah. Was considering moving this to a presenter object, but figured it might be overkill. I'd rather have this here now, than pick the wrong abstraction. 😄